app/org/sagebionetworks/bridge/play/controllers/SurveyController.java

```diff
@@ -178,9 +193,12 @@ public Result deleteSurvey(String surveyGuid, String createdOnString) throws Exc
         Survey survey = surveyService.getSurvey(keys);
         verifySurveyIsInStudy(session, studyId, survey);
         
-        surveyService.deleteSurvey(survey);
+        if ("true".equals(physical) && user.isInRole(Roles.ADMIN)) {
+            surveyService.deleteSurveyPermanently(survey);
```

Comment:
What about a developer asking for a physical delete? Should that fall back to logical delete? Or should that throw Unauthorized? (And corresponding unit test.)